goMFT Update error

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

update in lxc container

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [ ] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

_No response_

### 📝 Provide a clear and concise description of the issue.

While updating the container i got htis error message in verbose mode:

`  ✔️   Stopped GoMFT
⠴ Updating GoMFT to 0.2.11curl: (22) The requested URL returned error: 400

[ERROR] in line 1459: exit code 0: while executing command "$@" >> "$LOGFILE" 2>&1`

### 🔄 Steps to reproduce the issue.

several times the same error

### ❌ Paste the full error output (if available).

```  ✔️   Stopped GoMFT
⠴ Updating GoMFT to 0.2.11curl: (22) The requested URL returned error: 400

[ERROR] in line 1459: exit code 0: while executing command "$@" >> "$LOGFILE" 2>&1```

### 🖼️ Additional context (optional).

_No response_

now more detailed ! goMFT is after update not starting any longer !

`Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
build-essential is already the newest version (12.9).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
  ✔️   Stopped GoMFT

> gomft@1.0.0 postinstall
> npm run build

⠇ Updating GoMFT to 0.2.11
> gomft@1.0.0 build
> node build.js

⠋ Updating GoMFT to 0.2.11Building Tailwind CSS...
⠏ Updating GoMFT to 0.2.11Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme

Rebuilding...

Done in 2852ms.
Build completed successfully!

added 123 packages, and audited 124 packages in 6s

30 packages are looking for funding
  run `npm fund` for details

1 moderate severity vulnerability

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
⠋ Updating GoMFT to 0.2.11Building Tailwind CSS...
⠇ Updating GoMFT to 0.2.11Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme

Rebuilding...

Done in 1588ms.
Build completed successfully!
⠴ Updating GoMFT to 0.2.11(✓) Complete [ updates=98 duration=384.923224ms ]
⠇ Updating GoMFT to 0.2.11# github.com/starfleetcptn/gomft/internal/db/migrations
internal/db/migrations/012a_recover_transfer_configs_rename.go:13:6: RecoverTransferConfigsRename redeclared in this block
        internal/db/migrations/011a_recover_transfer_configs_rename.go:13:6: other declaration of RecoverTransferConfigsRename
internal/db/migrations/012b_recover_notification_services_rename.go:13:6: RecoverNotificationServicesRename redeclared in this block
        internal/db/migrations/011b_recover_notification_services_rename.go:13:6: other declaration of RecoverNotificationServicesRename
internal/db/migrations/012c_recover_auth_providers_rename.go:13:6: RecoverAuthProvidersRename redeclared in this block
        internal/db/migrations/011c_recover_auth_providers_rename.go:13:6: other declaration of RecoverAuthProvidersRename
⠙ Updating GoMFT to 0.2.11
[ERROR] in line 62: exit code 0: while executing command $STD go build -o gomft`

--- a/ct/gomft.sh
+++ b/ct/gomft.sh
@@ -51,11 +51,8 @@ function update_script() {
     tar -xzf "$temp_file"
     cp -rf "GoMFT-${RELEASE}"/* /opt/gomft/
     cd /opt/gomft
-    rm -rf /opt/gomft/node_modules
-    $STD npm ci
-    $STD node build.js
-    $STD go mod download
-    $STD go get -u github.com/a-h/templ
+    $STD npm install
+    $STD npm run build
     $STD "$HOME"/go/bin/templ generate
     export CGO_ENABLED=1
     export GOOS=linux

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -47,19 +47,13 @@ msg_ok "Installed Node.js"
 msg_info "Setup ${APPLICATION} (Patience)"
 temp_file=$(mktemp)
 RELEASE=$(curl -fsSL https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
-curl -fsSL "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -o $temp_file
-tar -xzf $temp_file
-mv GoMFT-${RELEASE}/ /opt/gomft
+curl -fsSL "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -o "$temp_file"
+tar -xzf "$temp_file"
+mv GoMFT-"${RELEASE}"/ /opt/gomft
 cd /opt/gomft
-$STD npm ci
-$STD node build.js
-$STD go mod download
 $STD go install github.com/a-h/templ/cmd/templ@latest
-$STD go get -u github.com/a-h/templ
-$STD $HOME/go/bin/templ generate
-export CGO_ENABLED=1
-export GOOS=linux
-$STD go build -o gomft
+$STD "$HOME"/go/bin/templ generate
+$STD go build -o gomft main.go
 chmod +x /opt/gomft/gomft
 JWT_SECRET_KEY=$(openssl rand -base64 24 | tr -d '/+=')
 
@@ -83,7 +77,7 @@ EMAIL_USERNAME=smtp_username
 EMAIL_PASSWORD=smtp_password
 EOF
 
-echo "${RELEASE}" >/opt/${APPLICATION}_version.txt
+echo "${RELEASE}" >/opt/"${APPLICATION}"_version.txt
 msg_ok "Setup ${APPLICATION}"
 
 msg_info "Creating Service"
@@ -109,7 +103,7 @@ motd_ssh
 customize
 
 msg_info "Cleaning up"
-rm -f $temp_file
+rm -f "$temp_file"
 $STD apt-get -y autoremove
 $STD apt-get -y autoclean
 msg_ok "Cleaned"

